.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this is good, but only works on your computer because you use Visual Studio Code

Reply:
That's right. 

I known a lot from you.